cmd/app/serve.go

```diff
@@ -242,7 +243,7 @@ func runServeCmd(cmd *cobra.Command, args []string) {
 	httpServer.startListener()
 
 	prom := http.Server{
-		Addr:    ":2112",
+		Addr:    ":" + viper.GetString("metrics-port"),
```

Comment:
nit: `fmt.Sprintf(":%v", viper.GetString("metrics-port"))`

Reply:
Whats the difference?

Reply:
I think it's one less string allocation, also imo it's more idiomatic

Reply:
Alrighty 👍🏻

Reply:
yep, that is why was just a nit :)